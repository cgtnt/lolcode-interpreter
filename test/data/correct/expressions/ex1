HAI

VISIBLE SUM OF 3 AN 4!
VISIBLE DIFF OF 10 AN 3!
VISIBLE PRODUKT OF 2 AN 5!
VISIBLE QUOSHUNT OF 20 AN 4!
VISIBLE MOD OF 17 AN 5!
VISIBLE BIGGR OF 7 AN 9!
VISIBLE SMALLR OF 7 AN 9!

VISIBLE BOTH SAEM SUM OF PRODUKT OF 2 AN 3 AN QUOSHUNT OF 12 AN 4 AN 10!
VISIBLE DIFFRINT PRODUKT OF 2 AN 2 AN 5!
VISIBLE BOTH SAEM PRODUKT OF 3 AN 4 AN SUM OF 5 AN 7!
VISIBLE BOTH SAEM SMALLR OF SUM OF 2 AN 5 AN 10 AN 7!

KTHXBYE
