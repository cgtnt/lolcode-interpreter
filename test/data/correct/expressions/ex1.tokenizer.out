@1:T_BEGIN:HAI-@3:T_COMMAND_TERMINATOR:
-@3:T_WRITE_STDOUT:VISIBLE-@3:T_PLUS:SUM OF-@3:T_T_INT:3-@3:T_AND:AN-@3:T_T_INT:4-@3:T_BANG:!-@4:T_COMMAND_TERMINATOR:
-@4:T_WRITE_STDOUT:VISIBLE-@4:T_MINUS:DIFF OF-@4:T_T_INT:10-@4:T_AND:AN-@4:T_T_INT:3-@4:T_BANG:!-@5:T_COMMAND_TERMINATOR:
-@5:T_WRITE_STDOUT:VISIBLE-@5:T_TIMES:PRODUKT OF-@5:T_T_INT:2-@5:T_AND:AN-@5:T_T_INT:5-@5:T_BANG:!-@6:T_COMMAND_TERMINATOR:
-@6:T_WRITE_STDOUT:VISIBLE-@6:T_QUOTIENT:QUOSHUNT OF-@6:T_T_INT:20-@6:T_AND:AN-@6:T_T_INT:4-@6:T_BANG:!-@7:T_COMMAND_TERMINATOR:
-@7:T_WRITE_STDOUT:VISIBLE-@7:T_MOD:MOD OF-@7:T_T_INT:17-@7:T_AND:AN-@7:T_T_INT:5-@7:T_BANG:!-@8:T_COMMAND_TERMINATOR:
-@8:T_WRITE_STDOUT:VISIBLE-@8:T_MAX:BIGGR OF-@8:T_T_INT:7-@8:T_AND:AN-@8:T_T_INT:9-@8:T_BANG:!-@9:T_COMMAND_TERMINATOR:
-@9:T_WRITE_STDOUT:VISIBLE-@9:T_MIN:SMALLR OF-@9:T_T_INT:7-@9:T_AND:AN-@9:T_T_INT:9-@9:T_BANG:!-@11:T_COMMAND_TERMINATOR:
-@11:T_WRITE_STDOUT:VISIBLE-@11:T_EQUAL:BOTH SAEM-@11:T_PLUS:SUM OF-@11:T_TIMES:PRODUKT OF-@11:T_T_INT:2-@11:T_AND:AN-@11:T_T_INT:3-@11:T_AND:AN-@11:T_QUOTIENT:QUOSHUNT OF-@11:T_T_INT:12-@11:T_AND:AN-@11:T_T_INT:4-@11:T_AND:AN-@11:T_T_INT:10-@11:T_BANG:!-@12:T_COMMAND_TERMINATOR:
-@12:T_WRITE_STDOUT:VISIBLE-@12:T_NOT_EQUAL:DIFFRINT-@12:T_TIMES:PRODUKT OF-@12:T_T_INT:2-@12:T_AND:AN-@12:T_T_INT:2-@12:T_AND:AN-@12:T_T_INT:5-@12:T_BANG:!-@13:T_COMMAND_TERMINATOR:
-@13:T_WRITE_STDOUT:VISIBLE-@13:T_EQUAL:BOTH SAEM-@13:T_TIMES:PRODUKT OF-@13:T_T_INT:3-@13:T_AND:AN-@13:T_T_INT:4-@13:T_AND:AN-@13:T_PLUS:SUM OF-@13:T_T_INT:5-@13:T_AND:AN-@13:T_T_INT:7-@13:T_BANG:!-@14:T_COMMAND_TERMINATOR:
-@14:T_WRITE_STDOUT:VISIBLE-@14:T_EQUAL:BOTH SAEM-@14:T_MIN:SMALLR OF-@14:T_PLUS:SUM OF-@14:T_T_INT:2-@14:T_AND:AN-@14:T_T_INT:5-@14:T_AND:AN-@14:T_T_INT:10-@14:T_AND:AN-@14:T_T_INT:7-@14:T_BANG:!-@16:T_COMMAND_TERMINATOR:
-@16:T_END:KTHXBYE-@17:T_COMMAND_TERMINATOR:
-@17:T_EOF:
